depthai_examples/src/stereo_inertial_publisher.cpp

```diff
@@ -0,0 +1,277 @@
+    const std::string rightPubName = rectify?std::string("right/image_rect"):std::string("right/image_raw");
+
+    dai::rosBridge::ImuConverter imuConverter(deviceName +"_imu_frame");
+
+    dai::rosBridge::BridgePublisher<sensor_msgs::Imu, dai::IMUData> ImuPublish(imuQueue,
+                                                                                     pnh, 
+                                                                                     std::string("imu"),
+                                                                                     std::bind(&dai::rosBridge::ImuConverter::toRosMsg, 
+                                                                                     &imuConverter, 
+                                                                                     std::placeholders::_1,
+                                                                                     std::placeholders::_2) , 
+                                                                                     30,
+                                                                                     "",
+                                                                                     "imu");
+
+    ImuPublish.addPubisherCallback();
+    
+     if(enableDepth){
+        std::cout << "In depth";
+        dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame> depthPublish(stereoQueue,
+                                                                                     pnh, 
+                                                                                     std::string("stereo/depth"),
+                                                                                     std::bind(&dai::rosBridge::ImageConverter::toRosMsg, 
+                                                                                     &rightconverter, // since the converter has the same frame name
+                                                                                                      // and image type is also same we can reuse it
+                                                                                     std::placeholders::_1, 
+                                                                                     std::placeholders::_2) , 
+                                                                                     30,
+                                                                                     rightCameraInfo,
+                                                                                     "stereo");
+        depthPublish.addPubisherCallback();
+        
+        if(depth_aligned){
+            auto imgQueue = device.getOutputQueue("rgb", 30, false);
+            dai::rosBridge::ImageConverter rgbConverter(deviceName + "_rgb_camera_optical_frame", false);
+            auto rgbCameraInfo = rgbConverter.calibrationToCameraInfo(calibrationHandler, dai::CameraBoardSocket::RGB, 1920, 1080); 
```

Comment:
when depth is aligned to rgb. CameraInfo of depth will be same as of rgb instead of rectified right

Reply:
I've tried to change that but this will cause a mismatch of the resolution between the depth cloud and the camera info on rviz. So, I'm currently keeping to use rightCameraInfo. Why is the original intention for changing this camera info?

Reply:
The reason why we need to change is because the intrinsics changes when we match the depth to rgb camera. So we need to use the rgb camera intrincs but rescaled version of it to match to the resolution of the depth. If this is not done point cloud conversion will be wrong.

Reply:
`auto depthCameraInfo = converter.calibrationToCameraInfo(calibrationHandler, dai::CameraBoardSocket::RGB, 1280, 720);` // This should work if the res of depth is 720p

Reply:
I think this is fixed in the current HEAD but I couldn't see the point cloud. The error is like the following.
`Error updating depth cloud: Depth image has invalid format (only 16 bit and float are supported)!`
This is reproduced by running stereo_node.launch, so I think this is another issue, not related to this commit.

Reply:
got it. I will check the pointcloud issue post merging this PR. it might be an issue from the Firmware.